### ISSUE TITLE
libsvg-cairo:  pthread-stubs was not found in the pkg-config 

### ISSUE THREAD

**Issue by tseemann** [Linuxbrew/homebrew-core#6931]
I'll try and look into this.

```
 brew install svg2png
==> Installing dependencies for svg2png: libsvg, libsvg-cairo
==> Installing svg2png dependency: libsvg
==> Downloading https://linuxbrew.bintray.com/bottles/libsvg-0.1.4_1.x86_64_linux.bottle.1.tar.gz
######################################################################## 100.0%
==> Pouring libsvg-0.1.4_1.x86_64_linux.bottle.1.tar.gz
  /home/linuxbrew/.linuxbrew/Cellar/libsvg/0.1.4_1: 14 files, 1.2MB
==> Installing svg2png dependency: libsvg-cairo
==> Downloading https://cairographics.org/snapshots/libsvg-cairo-0.1.6.tar.gz
######################################################################## 100.0%
==> ./configure --prefix=/home/linuxbrew/.linuxbrew/Cellar/libsvg-cairo/0.1.6_2
Last 15 lines from /home/linuxbrew/.cache/Homebrew/Logs/libsvg-cairo/01.configure:
checking if f95 supports -c -o file.o... yes
checking whether the f95 linker (/home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/shims/linux/super/ld -m elf_x86_64) supports shared libraries... yes
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking for ANSI C header files... (cached) yes
checking for X... libraries , headers
checking for gethostbyname... yes
checking for connect... yes
checking for remove... yes
checking for shmat... yes
checking for IceConnectionNumber in -lICE... yes
checking for pkg-config... /home/linuxbrew/.linuxbrew/opt/pkg-config/bin/pkg-config
checking for libsvg >= 0.1.2 cairo >= 0.5.0... Package pthread-stubs was not found in the pkg-config search path. Perhaps you should add the directory containing `pthread-stubs.pc' to the PKG_CONFIG_PATH environment variable Package 'pthread-stubs', required by 'xcb', not found
configure: error: Library requirements (libsvg >= 0.1.2 cairo >= 0.5.0) not met; consider adjusting the PKG_CONFIG_PATH environment variable if your libraries are in a nonstandard prefix so pkg-config can find them.

READ THIS: https://github.com/Linuxbrew/brew/wiki/troubleshooting
Please do not report this issue to Homebrew/brew or Homebrew/core, which support macOS only.
```

**Comment by iMichka** [comment]
This needs a `depends_on "linuxbrew/xorg/libpthread-stubs unless OS.mac?". Mind opening a PR for this?

**Comment by iMichka** [comment]
Will be fixed with #7197
